### ISSUE TITLE
Publish Docker Image to an registry

### ISSUE THREAD

**Issue by cryxnet** [MISP/x_old_misp_docker#169]
Please publish the Docker image to the designated registry like dockerhub. Its easier to deploy a whole stack,. This step is essential for distributing and deploying the application efficiently. By making the image available in the registry, it enables seamless collaboration and simplifies the deployment process. Thank you for your cooperation in completing this task.

**Comment by konsumate** [comment]
I agree - the container image should be all batteries included, instead of having to "build" it locally oneself.

**Comment by estebanbecker** [comment]
I totally agree
